src/rt/alloc.rs

```diff
@@ -15,6 +15,10 @@ pub(super) struct State {
 /// Track a raw allocation
 pub(crate) fn alloc(ptr: *mut u8) {
     rt::execution(|execution| {
+        if execution.log {
+            println!("alloc");
```

Comment:
Take it or leave it, but what do you think of 
```suggestion
            println!("alloc {:p}", ptr);
```
using the [`fmt::Pointer`](https://doc.rust-lang.org/std/fmt/trait.Pointer.html) formatter? Although printing the memory location doesn't provide a lot of information on its own, if we print it in `dealloc` as well, users can at least match pairs of `alloc` and `dealloc` calls.

Reply:
Done.